files/zh-cn/web/javascript/reference/statements/debugger/index.md

```diff
@@ -38,4 +42,4 @@ function potentiallyBuggyCode() {
 
 ## 参见
 
-- [在火狐开发者工具中调试](https://firefox-source-docs.mozilla.org/devtools-user/debugger/index.html)
+- [火狐浏览器 JavaScript 调试器¶](https://firefox-source-docs.mozilla.org/devtools-user/debugger/index.html) 在 Firefox 火狐浏览器源文档中
```

Comment:
I believe the symbol `¶` is copied by mistake, could you remove it from the en-US doc?

Addition, the current translation does not comply with Chinese grammar/customs, please improve the translation.